corehq/apps/export/forms.py

```diff
@@ -121,10 +121,13 @@ def __init__(self, has_form_export_permissions, has_case_export_permissions, *ar
                     ),
                     data_bind="visible: showAppType()",
                 ),
-                crispy.Field(
-                    'application',
-                    placeholder=_("Select Application"),
-                    data_bind="value: application",
+                crispy.Div(  # Case export fields
```

Comment:
This comment seems misleading - guessing it was accidentally copy/pasted from below.

Reply:
Removed